components/cam/src/physics/rrtmgp/radiation.F90

```diff
@@ -1422,6 +1448,12 @@ subroutine radiation_driver_sw(state, pbuf, cam_in, is_cmip6_volc, &
       call initialize_rrtmgp_fluxes(nday, nlev_rad+1, nswbands, fluxes_allsky_day, do_direct=.true.)
       call initialize_rrtmgp_fluxes(nday, nlev_rad+1, nswbands, fluxes_clrsky_day, do_direct=.true.)
 
+      ! Make sure temperatures are within range for aqua planets
```

Comment:
Just noting that this should be set via namelist now, since we have a `rrtmgp_clip_temperatures` option in SCREAM. I will make this change upstream in E3SM in a later PR though.